assets/src/js/plugins/form-template/utils.js

```diff
@@ -59,7 +73,7 @@ export const initializeIframeResize = function( iframe ) {
 
 				return false;
 			},
-			onInit: function( iframe ) {
+			onInit: function( ) {
```

Comment:
@henryholtgeerts do not remove this iframe param otherwise it can produce undesirable results.

ref: https://github.com/davidjbradshaw/iframe-resizer/blob/master/docs/parent_page/events.md#oninit

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/4642/commits/71aeca3dbc629ca2b5265863b17afaee0adbecb2